apps/portals/src/configurations/adknowledgeportal/synapseConfigs/experimental_tools.ts

```diff
@@ -1,7 +1,10 @@
 import { GenericCardSchema } from 'synapse-react-client/dist/containers/GenericCard'
 import { SynapseConstants } from 'synapse-react-client'
 import { SynapseConfig } from 'types/portal-config'
-import { CardConfiguration, LabelLinkConfig } from 'synapse-react-client/dist/containers/CardContainerLogic'
+import {
```

Comment:
I think the code style hasn't been enforced across all portals files yet, so lots of whitespace changes here